From 0a8fdd878458bef7ab555ebfad31eee9c0fcce44 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 27 Jun 2018 21:20:55 +0900
Subject: [PATCH 16/25] bus-util: make log level lower in
 request_name_destroy_callback()

Fixes #9442.

(cherry picked from commit 04e075b572b168a183cde29480dbb30d027d4fe7)
---
 src/shared/bus-util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/bus-util.c b/src/shared/bus-util.c
index 09e126d7f2..acbc98ad2a 100644
--- a/src/shared/bus-util.c
+++ b/src/shared/bus-util.c
@@ -1738,7 +1738,7 @@ static void request_name_destroy_callback(void *userdata) {
         assert(data);
         assert(data->n_ref > 0);
 
-        log_info("%s n_ref=%u", __func__, data->n_ref);
+        log_debug("%s n_ref=%u", __func__, data->n_ref);
 
         data->n_ref--;
         if (data->n_ref == 0)
-- 
2.19.1

